src/components/WorkspaceStarControl.ts

```diff
@@ -0,0 +1,99 @@
+import _ from 'lodash/fp';
```

Comment:
I believe that @msilva-broad would like us to have a "workspace" directory directly under `src`, similar to `src\data` and `src`analysis`.

Reply:
yes, tho it is easier to do that move as a seperate and very focused (only changing import paths) PR...

Reply:
But given that Blake is introducing a bunch of files, I figure we could at least put them in the new location (leaving older files alone).

Reply:
src/workspaces would be recommended when you make the move... it's not team-name. but feature-area, i.e. src/billing, etc.

Reply:
Why "workspaces" instead of "workspace"? Do we intend a different directory for things that are multiple workspaces (like the WorkspaceContainer) vs. a single workspaces (like the dashboard)?

Reply:
its whatever the team wants to call it for "workspace/s feature components, state and visual utils"...

Basically, what broad features are your team responsible for?  Workspace vs Workspaces is probably not relevant and should both live inside the new root folder.

Data related things like Ajax and Data-Providers are still to be in src/libs/ajax folder until we do some cleanup there.

Reply:
I personally would vote for `src/workspace`.

Reply:
I'm going to avoid adding the complication of moving around files in this PR, if that's ok with everyone.